components/alert/alert.css

```diff
@@ -0,0 +1,56 @@
+/* alert */
+.section-alert {
+    display: flex;
+    flex-direction: column;
+    gap: 10px; 
+}
+.alert{
```

Comment:
Only some part of the code is formatted. You can use Prettier to format the code and maintain consistency.

Reply:
Thanks for the suggestion.